pyproject.toml

```diff
@@ -109,6 +109,13 @@ dev = [
     "toolz",
     "tqdm>=4.30.0",
 ]
+docs = [
+    "mkdocs",
+    "mkdocs-material",
+    "mkdocstrings-python",
+    "mkdocs-gen-files",
+    "mkdocs-literate-nav",
+]
```

Comment:
psygnal should probably be pinned to version >=0.3.5
`_guiclass.py` imports evented from psygnal (and the psygnal docs suggest this was added in version 0.3.5 and not the currently pinned version 0.3.1)

* https://github.com/pyapp-kit/psygnal/releases/tag/v0.3.5
* https://github.com/pyapp-kit/psygnal/pull/83